### PR TITLE
Add BrowserSync

### PR DESCRIPTION
**Pull request by RobLoach**
https://github.com/kalamuna/kalastatic/issues/61

### CODE REVIEW

**Comment by madeofpeople** [comment]
Hey, why may we not want it?

**Comment by madeofpeople** [comment]
@RobLoach blew some of the dust off, merged to current conventions (a few little things had changed)... and it's working :D

**Comment by madeofpeople** [comment]
@robloach, this is tested and working, we dont quite have a story for it, but it is mergeable.

Thoughts?

**Comment by RobLoach** [comment]
@madeofpeople Cleaned up the PR, and removed the `connect` web server, since it's browsersync now. Code review passes for me, assigning to you to QA and merge.

File: Gruntfile.js [hunk 1]
```diff
@@ -41,7 +45,7 @@ module.exports = function(grunt) {
         options: {
           spawn: false,
           interupt: true,
-          livereload: true
+          livereload: false
```
Comment (madeofpeople) [inline Gruntfile.js]: Why livereload false?

**Pull request merged**